src/transactions/feed/TransactionDetailsScreen.tsx

```diff
@@ -104,23 +116,96 @@ function TransactionDetailsScreen({ navigation, route }: Props) {
       break
     case TokenTransactionTypeV2.SwapTransaction:
       content = <SwapContent exchange={transaction as TokenExchange} />
+      retryHandler = () => navigate(Screens.SwapScreenWithBack)
       break
   }
 
+  const transactionNetwork = networkConfig.networkIdToNetwork[transaction.networkId]
+  const openBlockExplorerHandler = transactionNetwork
+    ? () => {
+        navigate(Screens.WebViewScreen, {
+          uri: `${networkConfig.blockExplorerBaseTxUrl[transactionNetwork]}${transaction.transactionHash}`,
```

Comment:
nit: we're probably doing this all over the place, but I think we should join url paths using a function to properly handle cases where the string include `/` or not.

We have one such util from `@celo/utils`, but we could as well add our own. Since we want to eventually get rid of contractkit.

```suggestion
          uri: appendPath(networkConfig.blockExplorerBaseTxUrl[transactionNetwork]}, transaction.transactionHash),
```

Reply:
What if we import an NPM library for that, like [url-join](https://www.npmjs.com/package/url-join) to not reinvent the wheel?
What do you think @jeanregisser?

Reply:
since we are concatenating some appendix with base url, this perfectly fits to built-in [URL API](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL), switched to it